Tests/Marketplace/search_and_install_packs.py

```diff
@@ -36,7 +36,7 @@
 WLM_TASK_FAILED_ERROR_CODE = 101704
 
 GITLAB_SESSION = Session()
-CONTENT_PROJECT_ID = "2596"
+CONTENT_PROJECT_ID = os.getenv('CI_PROJECT_ID', '2596')  # the default is the id of the content repo in code.pan.run
```

Comment:
When the migration is over we'd need to change this default value.
(Add this to the Phase_2 issue.)